doc/README_2021.09.0_8.14+beta2.md

```diff
@@ -1,4 +1,4 @@
-# Coq Platform 2021.09.0 providing Coq 8.14+rc1 with a beta package pick
+# Coq Platform 2021.09.0 providing Coq 8.14.0 10/2021 with a beta package pick
```

Comment:
Not a big deal (and I don't want to be a bother) but my suggestion to put `(released Oct 2021)` and such also applied to the generated README titles.